video/out/wayland_common.c

```diff
@@ -2830,8 +2847,13 @@ static void set_color_management(struct vo_wayland_state *wl)
     wp_image_description_creator_params_v1_set_primaries_named(image_creator_params, primaries);
     wp_image_description_creator_params_v1_set_tf_named(image_creator_params, transfer);
 
+    if (pl_color_transfer_is_hdr(color.transfer))
+        pl_color_space_infer(&wl->target_params.color);
     struct pl_hdr_metadata hdr = wl->target_params.color.hdr;
-    if (wl->supports_display_primaries) {
+    bool use_metadata = hdr_metadata_valid(hdr);
```

Comment:
How about rejecting it completely we validate `max_cll` and `max_fall` and set it to 0? They are optional. max_luma/min_luma is not optional, but this after infer() should always be valid.